src/test/java/com/epam/tc/hw1/TestDiv.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import org.assertj.core.data.Percentage;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+
+public class TestDiv {
+    private static final double PRECISION = 0.0000001;
+
+    @DataProvider
+    public static Object[][] longData() {
+        return new Object[][]{
+                {20, 10, 2},
+                {5, 5, 1},
+                {100, 25, 4}
+        };
+    }
+
+    @DataProvider
+    public static Object[][] doubleData() {
+        return new Object[][]{
+                {5.7, 2, 2.85},
+                {3.3, 3, 1.1},
+                {5.6, 0.8, 7}
+        };
+    }
+
+    @Test(dataProvider = "longData")
+    public void divTestLong(long first, long second, long third) {
+
```

Comment:
Please, move your calculator instance creation into a separate class. You could call it, e.g., CalculatorTestBase or whatever. And there you could create some set up (and tear down) method with annotation @BeforeClass (@AfterClass) from TestNG.
Then your concrete test classes may be inherited from this base class.